src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -5381,36 +5381,26 @@ void UpdateSysSizing(EnergyPlusData &state, DataGlobalConstants::CallIndicator c
         // Correct the zone return temperature in ZoneSizing for the case of induction units. The calc in
         // ZoneEquipmentManager assumes all the air entering the zone goes into the return node.
         for (int CtrlZoneNum = 1; CtrlZoneNum <= state.dataGlobal->NumOfZones; ++CtrlZoneNum) {
-            if (!state.dataZoneEquip->ZoneEquipConfig(CtrlZoneNum).IsControlled) continue;
+            auto &zoneEquipConfig = state.dataZoneEquip->ZoneEquipConfig(CtrlZoneNum);
+            auto &zoneSizing = ZoneSizing(CurOverallSimDay, CtrlZoneNum);
```

Comment:
May as well eliminate the `ZoneSizing` array shortcut.

Reply:
Why? I see 8 occurances where this short cut is used. I was thinking 4 or more, use the short cut. zoneEquipConfig is used 3 times.

Reply:
I meant the (capital Z) `ZoneSizing` array shortcut, not the (lower z) `zoneSizing` object shortcut. I'm all for object shortcuts.

Reply:
I'm just slow. Yes, this will be eliminated everywhere.

    auto &ZoneSizing(state.dataSize->ZoneSizing);